clang/lib/Serialization/ASTWriterDecl.cpp

```diff
@@ -2529,12 +2531,12 @@ void ASTWriter::WriteDeclAbbrevs() {
   // VarDecl
   Abv->Add(BitCodeAbbrevOp(
       BitCodeAbbrevOp::Fixed,
-      21)); // Packed Var Decl bits:  Linkage, ModulesCodegen,
+      22)); // Packed Var Decl bits:  Linkage, ModulesCodegen,
```

Comment:
I'd expect this kind of change to also involve an update to `VERSION_MAJOR`, but it seems that doesn't happen much for Clang serialization changes? If it's not needed, that's fine.

Reply:
I don't know if we've ever written down a policy for this, and we've been inconsistent about it in the past.  In general, we don't try to read a pch/module if `getClangFullRepositoryVersion` doesn't match, and we check that in the same record that we check `VERSION_MAJOR`.  @Bigcheese or @jansvoboda11 any opinion?